Packs/CortexAttackSurfaceManagement/Scripts/GenerateASMReport/test_data/args.json

```diff
@@ -96,5 +96,6 @@
       "Source": "AWS"
     },
     { "Key": "env", "Value": "sbx", "Source": "AWS" }
-  ]
+  ],
+ "report_type": "summary"
```

Comment:
Is it worth having another one for `analysis`?

Reply:
might need to for coverage

Reply:
actually, didn't need it